data-raw/fxtas_tmp.R

```diff
@@ -0,0 +1,327 @@
+## pairwise comparisons for latent sub-types
+## compare sub-type proportions to overall proportion
+
+test_obj <- table_subtype_by_demographics(
+  patient_data,
+  subtype_and_stage_table =
+    results_cv_max$subtype_and_stage_table
+)
+
+# extract info for pairwise comparisons between sub-type and overall props
+## CGG
+names(test_obj$meta_data$df_stats) <- test_obj$meta_data$variable
```

Comment:
@mponzini I don't see the `$meta_data` object in `test_obj` when I try to run this code